src/services/directory.ts

```diff
@@ -36,24 +33,35 @@ export function fetchRootFolders(types: string[]) {
     });
 }
 
-export function fetchDirectoryContent(directoryUuid: UUID, types?: string[]) {
+export function fetchDirectoryContent(
+    directoryUuid: UUID,
+    types?: string[]
+): Promise<ElementAttributes[]> {
     console.info("Fetching Folder content '%s'", directoryUuid);
 
     // Add params to Url
     const urlSearchParams = getRequestParamFromList(
         types,
         'elementTypes'
     ).toString();
-    const fetchDirectoryContentUrl = `${PREFIX_DIRECTORY_SERVER_QUERIES}/v1/directories/${directoryUuid}/elements?${urlSearchParams}`;
+
+    let fetchDirectoryContentUrl = `${PREFIX_DIRECTORY_SERVER_QUERIES}/v1/directories/${directoryUuid}/elements`;
+    if (urlSearchParams.length > 0) {
```

Comment:
```suggestion
    if (urlSearchParams) {
```
It's a string no ?

Reply:
It's a string once toString has been called but has is it is an object like a Map, but I just tried out, if urlSearchParams is empty `if (urlSearchParams) {` returns true but `if (urlSearchParams.length > 0) {` doesn't work either, so I guess the correct condition would be `if (urlSearchParams.toString()) {`

Reply:
I don't get it, we need to call toString() two times ?